deploy/kubernetes/images/prow-gke-release-staging-rc/image.yaml

```diff
@@ -25,7 +25,7 @@ metadata:
 imageTag:
   name: gke.gcr.io/csi-resizer
   newName: gcr.io/gke-release-staging/csi-resizer
-  newTag: "v1.0.1-gke.0"
+  newTag: "v0.5.0-gke.0"
```

Comment:
can we keep newer resizer and snapshotter? The new versions have bug fixes that helped deal with test flakes.

Reply:
Done